internal/pkg/model/mapper_test.go

```diff
@@ -0,0 +1,67 @@
+package model
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
+)
+
+func TestObjectFiles(t *testing.T) {
```

Comment:
Unit testy ukazuju ako to funguje. Subory je mozne tagovat a na zaklade tagov filtrovat.